src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs

```diff
@@ -31,11 +32,28 @@ public void ItemStackPanel_MouseDown(object sender, RoutedEventArgs e)
             viewModel.Model.IsExpanded = !viewModel.Model.IsExpanded;
```

Comment:
For consistency, please rename the following event handlers to:

`OnShowContextMenuFromLeftClicked`
`OnSortButtonClicked`
`OnInstallLatestButtonDropDownClicked`
`OnInstallVersionButtonDropDownClicked`